docs/source/community/communication.md

```diff
@@ -75,8 +75,7 @@ We will send a summary of all sync ups to the dev@datafusion.apache.org mailing
 Our source code is hosted on
 [GitHub](https://github.com/apache/arrow-datafusion). More information on contributing is in
 the [Contribution Guide](https://github.com/apache/arrow-datafusion/blob/master/CONTRIBUTING.md)
-, and we have curated a [good-first-issue]
-(https://github.com/apache/arrow-datafusion/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
+, and we have curated a [good-first-issue](https://github.com/apache/datafusion-ballista/labels/good%20first%20issue)
```

Comment:
GitHub provides a `/contribute` url that lists good first issues:

```suggestion
, and we have curated a [good-first-issue](https://github.com/apache/datafusion-ballista/contribute)
```

Reply:
Didn't know it, thank you, merged.